sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
 
+To fill this form:
```

Comment:
I feel like these instructions should be in a separate file? They're very long and are unlikely to be useful except maybe once, so having this be so prominent in the readme might push down other valuable content like install instructions. Really really want install instructions "above the fold".

Reply:
Anyone else have feelings either way here?

Reply:
That seems fair, then just have a link to that separate file here.

Reply:
I moved them to a separate file!